employeeSearchResults.php

```diff
@@ -46,21 +48,22 @@
 
 
 if(isset($_GET['submit'])){
-    //KW capture and store user inputs in a variables.
+    //KW capture and store user inputs in a variabl
     $empFirstName = htmlspecialchars($_GET['firstName']);
     $empLastName = htmlspecialchars($_GET['lastName']);
     $searchLimit = $_GET['limitSearchResults'];
     $resultOrder = $_GET['ascOrDesc'];
     $orderByLastOrFirstName = $_GET['orderResultsBy'];
-    //Assign values to variable and run a query.
+    //Assign values to variable and run
     echo "You clicked submit";
     $query = "SELECT * FROM employee WHERE FIRST_NAME LIKE '%$empFirstName%';";
```

Comment:
instead of a regular sql select, you should utilise mysqli prepared statements to handle the security side of things.
```
$order_by = $_GET['orderBy'] ?? $_GET['orderBy'] ?? FIRST_NAME;
$mysqli= new mysqli('localhost','NEVERSAVEYOURCONNECTIONDATAINSOURCECONTROL','','');
$stmt = $mysqli->prepare("
SELECT * FROM employee WHERE FIRST_NAME LIKE ?
ORDER BY ${order_by} ASC
");
$stmt->bind_param("s", $_GET['firstName']);
$stmt->execute();
$result = $stmt->get_result();
$stmt->close();
```

Reply:
That is a good way. And doing it that way I believe you can pass an array of params. I suggest you checkout mysqli docs